test/conformance/route_test.go

```diff
@@ -100,21 +100,19 @@ func TestRouteCreation(t *testing.T) {
 	//add test case specific name to its own logger
 	logger := logging.GetContextLogger("TestRouteCreation")
 
-	var imagePaths []string
-	imagePaths = append(imagePaths, test.ImagePath(pizzaPlanet1))
-	imagePaths = append(imagePaths, test.ImagePath(pizzaPlanet2))
-
 	var objects test.ResourceObjects
-	var names test.ResourceNames
-	names.Config = test.AppendRandomString("test-route-creation-", logger)
-	names.Route = test.AppendRandomString("test-route-creation-", logger)
-	names.TrafficTarget = test.AppendRandomString("test-route-creation-", logger)
+	names := test.ResourceNames{
+		Config:        test.AppendRandomString("test-route-creation-", logger),
```

Comment:
We seem to have a pattern throughout our tests of creating new random suffixes for our different resource names. Do you have any thoughts on using the same suffix (or even same name) across resources to easily correlate objects for a single test run?

Reply:
No, I'm also not sure why we don't just use `generateName:`